Procfile

```diff
@@ -0,0 +1,2 @@
+release: ENV_SILENT=true node ace migration:run --force
+web: ENV_SILENT=true npm start
```

Comment:
Regarding `ENV_SILENT`: https://adonisjs.com/docs/4.0/configuration-and-env#_disabling_env_file